firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/NewReleaseFetcherTest.java

```diff
@@ -118,6 +118,7 @@ public void setup() {
             .setApplicationInfo(applicationInfo)
             .build();
     packageInfo.setLongVersionCode(INSTALLED_VERSION_CODE);
+    packageInfo.versionName = "1.0";
```

Comment:
Please add new test cases that cover the new functionality: version name is different, version name is the same, etc.

Reply:
When the version name is the same it should not return a new release which we have tests for. Will add one for when version name is different